Safety/AttitudeManager/Src/attitudeStateClasses.cpp

```diff
@@ -112,8 +130,13 @@ void OutputMixingMode::execute(attitudeManager* attitudeMgr)
 
     if (ErrorStruct.errorCode == 0)
     {
-        attitudeMgr->setState(sendToSafetyMode::getInstance());
-    }
+        // setting PWM channel values
+        pwm.set(FRONT_LEFT_MOTOR_CHANNEL, PID_Output_t -> frontLeftPercent);
+        pwm.set(FRONT_RIGHT_MOTOR_CHANNEL, PID_Output_t -> frontRightPercent);
+        pwm.set(BACK_LEFT_MOTOR_CHANNEL, PID_Output_t -> backLeftPercent);
+        pwm.set(BACK_RIGHT_MOTOR_CHANNEL, PID_Output_t -> backRightPercent);
```

Comment:
sets the PWM values for each of the channels based on the declared and set PWMChannels